src/internal.rs

```diff
@@ -115,10 +115,12 @@ pub(crate) struct Rav1dContext_frame_thread {
     pub next: c_uint,
 }
 
+pub type GrainLut<Entry> = [[Entry; GRAIN_WIDTH]; GRAIN_HEIGHT + 1];
```

Comment:
Is this typedef meant to be used somewhere?

Reply:
Yeah it's used in a bunch of the filmgrain signatures.